### PR TITLE
Rebrand posts frontend

### PR DESCRIPTION
**Pull request by Mikitsik**
- Updated some node dependencies.
- Fixed pages/posts HTML & CSS.

### CODE REVIEW

File: .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I"�
```
Comment (ka8725) [inline .jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910:1]: I think this file is not needed in git. Add this folder to `.gitingnore`?

**Comment by ka8725** [comment]
Post title is too small. font-size should be 40px?

File: Rakefile.rb [hunk 2]
```diff
@@ -34,12 +34,50 @@
...
     post.puts "tags: [#{tags}]"
+    post.puts "featured_post: false"
     post.puts "comments: true"
     post.puts "share: true"
+    post.puts "featured_post: false"
     post.puts "---"
   end
 end
 
+# usage rake featured_post
```
Comment (ka8725) [inline Rakefile.rb:45]: Thanks for this Rake task! Ideally, this comment should be in README.

File: _posts/2012-03-02-do-not-give-name-test-for-your-models.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ title: "Do not give name Test for your models"
 description: "Ruby on Rails guides for strange situation with name Test for models. There are many reasons to do not give name Test for your models in Ruby On Rails. Validates uniqueness in scope test_id does not work. There is a already Test module in Ruby On Rails"
 tags: [ruby, rails, active record, reserved_words]
 share: true
+featured_post: false
```
Comment (ka8725) [inline _posts/2012-03-02-do-not-give-name-test-for-your-models.md:7]: it's better not to have this option for all posts, and only featured one can have it with value = `true`.

Comment (Mikitsik) [reply _posts/2012-03-02-do-not-give-name-test-for-your-models.md:7]: Ok. I've done it.

File: Rakefile.rb (hunk 2 above)
Comment (Mikitsik) [reply Rakefile.rb:45]: Ok. I've added.

**Pull request merged**